certora/helpers/DummyERC20Impl.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity ^0.8.0;
+
+// with mint
+contract DummyERC20Impl {
+    uint256 t;
+    mapping (address => uint256) b;
+    mapping (address => mapping (address => uint256)) a;
+
+    string public name;
+    string public symbol;
+    uint public decimals;
```

Comment:
the ERC says decimals should be `uint8`, not `uint`

Reply:
Will update or just use OZ implementation.